src/PotentialTypes/BasisPotentials/force.jl

```diff
@@ -0,0 +1,11 @@
+function force(A::AbstractSystem, p::BasisPotential)
+    number_of_particles = length(A)
+    d = evaluate_basis_d(A, p.basis_params)
+    f = [zeros(3) for i = 1:number_of_particles]
+    for (i, di) in enumerate(d)
+        # Note that we only take f[i] += ... 
+        # This is because the descriptors should already account for f[j]
+        f[i] += dot(di, p.coefficients)
+    end
+    return SVector{number_of_particles}([SVector{3}(fi) for fi in f])
```

Comment:
You may have noticed that in the other force file I changed the returned outer array to not be static. Here is a quote from the StaticArrays documentation that informed this decision ([source](https://github.com/JuliaArrays/StaticArrays.jl#speed)):

> Note that in the current implementation, working with large StaticArrays puts a lot of stress on the compiler, and becomes slower than Base.Array as the size increases. A very rough rule of thumb is that you should consider using a normal Array for arrays larger than 100 elements.

I'd suggest changing this line to `SVector{3}.(f)`.

This is actually a more compact version that what I thought of last night. You could change the other force file to be the same if you feel like it.

Reply:
Good idea! I have replaced the corresponding lines.